salt_stack/salt/lambda-dev/files/makedomainenv

```diff
@@ -86,22 +86,7 @@ else
    zip -r9 ~/lambdazips/multilambda.${1}.zip *
 
    cd ~/virtualenvs/${1}/local/lib/python3.6/site-packages/lambda
-   zip -g ~/lambdazips/multilambda.${1}.zip lambda_loader.py
-   zip -g ~/lambdazips/multilambda.${1}.zip write_s3_index_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip fanout_write_id_index_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip write_id_index_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip write_index_failed_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip fanout_enqueue_cuboid_keys_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip index_find_cuboids_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip batch_enqueue_cuboid_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip fanout_dequeue_cuboid_keys_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip dequeue_cuboid_keys_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip get_num_msgs_cuboid_keys_queue_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip check_for_index_throttling_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip invoke_index_supervisor_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip split_cuboids_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip load_ids_from_s3_lambda.py
-   zip -g ~/lambdazips/multilambda.${1}.zip start_sfn_lambda.py
+   zip -g ~/lambdazips/multilambda.${1}.zip lambda_loader.*
```

Comment:
I don't remember seeing where `lambda_loader.*` was introduced

Reply:
Very good catch. This is of course not the actual makedomainenv running on the lambda build server since we don't rebuild it regularly.